src/components/MediaLink/BlipFile.vue

```diff
@@ -121,18 +121,35 @@ export default {
   .bubble {
     &.left, &.middle {
       .description-wrapper {
-        color: $vue-gray;
+        color: $color-content-default;
       }
     }
 
     &.right {
       .file-icon-wrapper {
-        background-color: $vue-white;
+        background-color: $color-surface-3;
         padding-right: 20px !important;
+        border-radius: 13px 0 0 13px;
       }
       .description-wrapper {
-        color: $vue-white;
+        background-color: $color-surface-2;
+        color: $color-content-default;
         padding-left: 20px !important;
+        border-radius: 0 2px 13px 0;
+      }
+    }
+
+    &.left {
+      .file-icon-wrapper {
+        background-color: $color-surface-2;
+        padding-right: 20px !important;
+        border-radius: 13px 0 0 2px;
+      }
+      .description-wrapper {
+        background-color: $color-surface-1;
+        color: $color-content-default;
+        padding-left: 20px !important;
+        border-radius: 0 13px 13px 0;
```

Comment:
Should the `background color` be different between left and right?

Reply:
Yes it should. It was a definition of the design team.